components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/resources/dbscripts/identity.sql

```diff
@@ -532,6 +532,15 @@ CREATE TABLE IF NOT EXISTS IDP (
   UNIQUE (UUID)
 );
 
+CREATE TABLE IF NOT EXISTS SP_IDP_ATTR (
+            IDP_ID INTEGER NOT NULL,
```

Comment:
do we need a new ID ?

Reply:
We do not have an auto incrementing ID for tables which have a composite primary key